### DISCUSSION TITLE
Cloud Gateway Max IPTV KPN issues

### DISCUSSION THREAD

**Discussion by ernstjandevries1** [fabianishere/udm-iptv#353] [Q&A]
Hi all,

I have a Cloud Gateway Max directly on my fiber connection, so no modem of a provider anymore. 
My Provider is KPN. My setup is the same as in the documentation
Internet is working fine now with my Cloud Gateway Max and AP's.

I followed the steps of Fabian's script to get IPTV working, but without succes. The TV screens keeps loading at 85% and gives an error afterwards. 

Are there any specific settings I need to enter in the config file? For example, my current file looks like this:

IPTV_WAN_INTERFACE="eth4"
IPTV_WAN_VLAN="4"
IPTV_WAN_VLAN_INTERFACE="TV"
IPTV_WAN_RANGES="213.75.0.0/16  217.166.0.0/16"
IPTV_WAN_DHCP_OPTIONS="-O staticroutes -V IPTV_RG"
IPTV_LAN_INTERFACES=""
IPTV_IGMPPROXY_PROGRAM="improxy"
IPTV_IGMPPROXY_DISABLE_QUICKLEAVE="false"
IPTV_IGMPPROXY_DEBUG="true"
IPTV_IGMPPROXY_IGMP_VERSION="2"

I already changed eth8 to eth4 but that didn't make it work as well. 
Not sure what I need to add at "IPTV_LAN_INTERFACES="" " Any advice? 

Service log: 
=== Service Logs ===
Oct 11 14:12:39 Cloud-Gateway-Max udm-iptvd[75171]: Device eth4 for TV does not exist
Oct 11 14:12:39 Cloud-Gateway-Max systemd[1]: udm-iptv.service: Main process exited, code=exited, status=1/FAILURE
Oct 11 14:12:39 Cloud-Gateway-Max systemd[1]: udm-iptv.service: Failed with result 'exit-code'.
Oct 11 14:12:44 Cloud-Gateway-Max systemd[1]: udm-iptv.service: Scheduled restart job, restart counter is at 414.
Oct 11 14:12:44 Cloud-Gateway-Max systemd[1]: Stopped IPTV support for the UniFi Dream Machine.
Oct 11 14:12:44 Cloud-Gateway-Max systemd[1]: Started IPTV support for the UniFi Dream Machine.
Oct 11 14:12:44 Cloud-Gateway-Max udm-iptvd[75274]: Device "eth8" does not exist.
Oct 11 14:12:44 Cloud-Gateway-Max udm-iptvd[75271]: Device eth8 for TV does not exist
Oct 11 14:12:44 Cloud-Gateway-Max systemd[1]: udm-iptv.service: Main process exited, code=exited, status=1/FAILURE
Oct 11 14:12:44 Cloud-Gateway-Max systemd[1]: udm-iptv.service: Failed with result 'exit-code'.

Also those are my settings in my Cloud Gateway Max:

**Internet with PPPoE**

![Scherm­afbeelding 2024-10-11 om 14 18 32](https://github.com/user-attachments/assets/aca66722-bedc-4906-97a1-a9b2fecbb7b6)

**Network:** 
With Snooping enabled
TV is the VLAN I need to use for TV

![Scherm­afbeelding 2024-10-11 om 14 19 27](https://github.com/user-attachments/assets/d7ab960d-3bb6-41b7-9820-f1322ae3066c)
![Scherm­afbeelding 2024-10-11 om 14 19 36](https://github.com/user-attachments/assets/b53b2eb1-1039-410e-903f-d393778f28cb)

Anyone any idea on how to solve this? Or any additional settings needed?

**Comment by ThijsvanRijn** [comment]
After the recent UniFi network update, my TV also stopped working, and I realized I forgot to disable the auto-update feature. So, I’m not sure whether the issue is on your end or if we need an update to the script.

**Comment by ernstjandevries1** [comment]
If found the issue. It was because the following setting was missing:

IPTV_LAN_INTERFACES

if have set that one to "br661" and now all is working fine.
